### PR TITLE
Several refinements.

### PR DESCRIPTION
**Pull request by nickhsine**
### KanbanFlow Card
[[story-3] Refines the final styles.](https://kanbanflow.com/t/wav5Ty2t)

### CODE REVIEW

**Comment by nickhsine** [comment]
@YuCJ This commit becomes so big. 
Please help to review this PR as soon as possible.
Thanks.

File: src/constants/prop-types/article-page.js [hunk 1]
```diff
@@ -0,0 +1,21 @@
+import PropTypes from 'prop-types'
+
+const elementColors = PropTypes.shape({
+  primary: PropTypes.shape({
+    text: PropTypes.string,
+    accent: PropTypes.string,
+    support: PropTypes.string,
+    background: PropTypes.string,
+  }),
+  base: PropTypes.shape({
```
Comment (YuCJ) [inline src/constants/prop-types/article-page.js:16]: missing `secondary` prop types

File: src/components/aside/tools.js [hunk 2]
```diff
@@ -42,6 +54,59 @@ const ToolsBlock = styled.div`
   `}
 `
 
+function changeFontSizeOffsetToPect(fontSizeOffset) {
```
Comment (YuCJ) [inline src/components/aside/tools.js]: `Percentage` or `Pct` as abbreviation

File: src/components/body/multimedia.js [hunk 3]
```diff
@@ -0,0 +1,155 @@
...
+    position: relative;
+
+    & > figure, & > iframe {
+      padding: 0;
+      border: 0;
+      margin: 0;
+    }
+
+    ${mq.tabletAndBelow`
+      // 10px is the left border width of body
```
Comment (YuCJ) [inline src/components/body/multimedia.js]: I thought `//` will not work in template literal and CSS, but it seems works fine. 👀

**Review by YuCJ — COMMENTED** [review]
LGTM 👍

File: src/components/body/multimedia.js (hunk 3 above)
Comment (nickhsine) [reply src/components/body/multimedia.js]: I am not sure either.
However, I will change `//` to `/**/` instead.

Comment (nickhsine) [reply src/components/body/multimedia.js]: It seems that `styled-components` supports line comments.
https://github.com/styled-components/stylelint-processor-styled-components/issues/156

**Pull request merged**